NitroxServer/ConsoleCommands/KickCommand.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using NitroxServer.ConsoleCommands.Abstract;
+using NitroxServer.GameLogic;
+using NitroxModel.Packets;
+using NitroxModel.Logger;
+
+namespace NitroxServer.ConsoleCommands
+{
+    class KickCommand : Command
+    {
+        private PlayerManager playerManager;
+
+        public KickCommand(PlayerManager playerManager)
+        {
+            Name = "kick";
+            this.playerManager = playerManager;
+        }
+
+        public override void RunCommand(string[] args)
+        {
+            try
+            {
+                Player player = playerManager.GetPlayers().Where(t => t.Name == args[0]).Single();
+                args = args.Skip(1).ToArray();
+                player.SendPacket(new UserKicked("You were kicked from the server! \n Reason: " + string.Join(" ", args)));
```

Comment:
Do other players receive a disconnect packet?

Reply:
Yes tested on multiple occasions(currently implemented on my own servers, have had some people join using this for testing purposes while others were with current release)

Reply:
Properly gives the kicked player the disconnected message(default one not the kicked message) and notifies other clients he has disconnected